backend/src/test/resources/org/cryptomator/hub/flyway/V9999__Test_Data.sql

```diff
@@ -65,22 +65,68 @@ VALUES
 
 INSERT INTO "audit_event" ("id", "timestamp", "type")
 VALUES
-    (999, '1970-01-01T00:00:02.900Z', 'CREATE_VAULT'),
-    (1000, '1970-01-01T00:00:03Z', 'UPDATE_VAULT_MEMBERSHIP'),
-    (1001, '1970-01-01T00:00:04Z', 'UPDATE_VAULT_MEMBERSHIP'),
-    (1111, '1970-01-01T00:00:04.999Z', 'UNLOCK_VAULT'),
-    (4242, '1970-01-01T00:00:05Z', 'UNLOCK_VAULT');
+    (10, '2020-02-20T20:20:20.010Z', 'CREATE_VAULT'),
+    (11, '2020-02-20T20:20:20.011Z', 'UPDATE_VAULT_MEMBERSHIP'),
+    (12, '2020-02-20T20:20:20.012Z', 'UPDATE_VAULT_MEMBERSHIP'),
```

Comment:
The reason to directly start with quite high numbers is, that also preceding tests generate audit events and they might then fail, because an audit event with sequence number 10 already exists. My suggestion is to directly start at 1000, or at least 100.

Reply:
That's why I added this 16915815d10b17ce8bbb5464c39a6083dcf9d830.